### ISSUE TITLE
VideoTexture keeps accessing camera after being disposed

### ISSUE THREAD

**Issue by PrimaryFeather** [Gamua/Adobe-Runtime-Support#3]
## Problem Description

Even after disposing a VideoTexture that has a camera attached to it, the camera is still being accessed.

Tested with iOS 10.3.2, compiled with AIR SDK 26.0.0, build 112.

Adobe Tracker Issue: https://tracker.adobe.com/#/view/AIR-4198123

## Steps to Reproduce

* Compile & run the class [VideoTextureBug.as](https://gist.github.com/PrimaryFeather/e3fa56ef341e0f246f6bb42831b3ef38) as Flash Project.
* Allow the camera to be accessed when the dialog pops up.
* A spinning triangle with the camera contents will appear.
* When hitting "x" on the keyboard, the texture will be disposed.

Actual Result: The light on the camera keeps glowing.  
Expected Result: The light on the camera should go out.

In other words: when disposing the texture, any connection to the camera should be severed.

## Known Workarounds

Before disposing the video texture, call `videoTexture.attachCamera(null)` manually.

**Comment by PrimaryFeather** [comment]
Adobe fixed this problem with AIR 27.0.116. Thanks a lot for your efforts!
